docs/content/docs/ops/state/state_backends.md

```diff
@@ -150,18 +188,29 @@ If you want to use the `EmbeddedRocksDBStateBackend` in your IDE or configure it
 </dependency>
 ```
 
+Same for `ForStStateBackend`:
+```xml
+<dependency>
+    <groupId>org.apache.flink</groupId>
+    <artifactId>flink-statebackend-forst</artifactId>
+    <version>{{< version >}}</version>
+    <scope>provided</scope>
+</dependency>
+```
+
 {{< hint info >}}
-Since RocksDB is part of the default Flink distribution, you do not need this dependency if you are not using any RocksDB code in your job and configure the state backend via `state.backend.type` and further [checkpointing]({{< ref "docs/deployment/config" >}}#checkpointing) and [RocksDB-specific]({{< ref "docs/deployment/config" >}}#rocksdb-state-backend) parameters in your [Flink configuration file]({{< ref "docs/deployment/config#flink-configuration-file" >}}).
+Since RocksDB and ForSt is part of the default Flink distribution, you do not need this dependency if you are not using any RocksDB code in your job and configure the state backend via `state.backend.type` and further [checkpointing]({{< ref "docs/deployment/config" >}}#checkpointing) and [RocksDB-specific]({{< ref "docs/deployment/config" >}}#rocksdb-state-backend) or [ForSt-specific]({{< ref "docs/deployment/config" >}}#forst-state-backend) parameters in your [Flink configuration file]({{< ref "docs/deployment/config#flink-configuration-file" >}}).
```

Comment:
nits:
 is part -> are part
this dependency -> these dependancies
any RocksDB code -> any RocksDB or ForSt code